src/keycloak/keycloak_admin.py

```diff
@@ -1349,21 +1349,24 @@ def get_subgroups(self, group, path):
         # went through the tree without hits
         return None
 
-    def get_group_children(self, group_id):
-        """Get group children by id.
+    def get_group_children(self, parent_group_id, query=None):
```

Comment:
I agree with the naming, unfortunately, this change would make this a breaking change for users relying on the argument being named `group_id`, thus making an upgrade for them to break the working functionality.

Python-keycloak version 4.X will come with a couple more breaking changes than just this one. I'd suggest either keeping the argument name the same (group_id) or creating a new method and adding a deprecated warning to this one.

I'm also open to hearing other suggestions of how to resolve this (perhaps merging this into release/4-like branch, etc.)

Reply:
I will change the parameter name back to group_id.